pyproject.toml

```diff
@@ -6,3 +6,7 @@ multi_line_output = 3
 show_error_codes = true
 ignore_missing_imports = true
 no_warn_no_return = true
+
+[[tool.mypy.overrides]]
+module = "tests.po_lib_to_return.*"
+ignore_errors = true
```

Comment:
Could you please add a comment, explaining why these errors are ignored?

Reply:
Updated in https://github.com/scrapinghub/web-poet/pull/84/commits/88c511d51cfdf64f1b4ae69563f7a97f8cf61cff.